my-vue3-app/src/store/reports/actions.ts

```diff
@@ -1,21 +1,56 @@
 import { ActionContext } from 'vuex';
 import api from '@/services/api';
-import { Pagination } from './state';
-import { SET_DATA, SET_PAGINATION, SET_LOADING } from './mutations';
+import {
+  PAGINATION, Pagination, Reports, REPORTS, State,
+} from './state';
+import {
+  ADD_DATA, SET_PAGINATION, SET_LOADING,
+} from './mutations';
 
 export const FETCH_REPORTS = 'load_reports';
 
+const applyFilterSortToReports = (
+  pagination: Pick<Pagination, 'order'>,
```

Comment:
what is Pick?

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys